compose/cli/docopt_command.py

```diff
@@ -34,6 +34,10 @@ def parse(self, argv):
         if docstring is None:
             raise NoSuchCommand(command, self)
 
+        # checks to see if the command is help and if so outputs top level command options
+        if command == 'help' and options['ARGS'] == []:
+            raise SystemExit(docstring + '\n-------------------\n\n' + command_help)
```

Comment:
I'm not familiar with python, but this looks as if an error will be thrown, setting the exit code to non-zero and writing to stderr.
As `docker-compose help` is a legal invocation and not an error condition, the command's exit code should be 0 and the output should go to stdout.
Can you please verify this?